woocommerce/woocommerce-android
# Make payment section always visible

Summary
==========
Fixes issues #5908 and #5907 by making the payment section always visible and adjusting the Fee percentage mode to be available only when the Order contains any price value as total beside the fee itself. Since both issues scenarios are tied together, I decided to tackle everything in a single PR testing all at once.

Screenshots
==========
https://user-images.githubusercontent.com/5920403/156269729-6f39057f-f3c5-462c-83df-b1242ac83125.mp4

How to Test
==========
1. Make sure the feature flag `ORDER_CREATION_M2` is enabled.
2. Open order creation form.
3. Notice the Payment section is now visible from the start
4. Notice the button "Add fee" on the payment section and click on it.
5. Once inside the fee view, notice the `percentage fee` toggle is gone since there's no valid order total
6. Enter a fee amount and hit `done`.
7. Now back at the New Order view, add a product to the order and enter the `Fee` view back again
8. Notice that the `percentage fee` toggle is now visible


Update release notes:
